source/optional_features/digital_wallets/index.html.md.erb

```diff
@@ -15,6 +15,10 @@ Sign in to the [GOV.UK Pay admin tool](https://selfservice.payments.service.gov.
 
 In your __Settings__, select the __Digital wallet__ tab in __Card types__. In the next screen, enable Apple Pay.
 
+You do not need to ask Worldpay to enable Apple Pay on your Worldpay account
+to use this feature, because GOV.UK Pay fully handles the Apple Pay
```

Comment:
```suggestion
, because GOV.UK Pay fully handles the Apple Pay
```

Just for conciseness, as I think the meaning is still clear without this.

Reply:
I know this is a familiar theme from me, but... I think we can lose everything from "because". That's not because the reason isn't important - it's because the reason is fully implied through just the instruction. We can say just "You do not need to ask Worldpay to enable Apple Pay in your Worldpay account", and the clear implication is "because it will work without you doing that".